src/App.jsx

```diff
@@ -26,6 +27,7 @@ const App = () => {
   }, []);
 
   const loggedIn = useSelector((state) => state.login.isLoggedIn);
+  const role = useSelector((state) => state.login.role)
```

Comment:
- [ ] [Optional] You can short those lines of code by using object deconstruction, something like this:

``` javascript
const { isLoggedIn, role } = useSelector((state) => state.login)
```

Reply:
Good idea! I've changed that 👍